apps/native-component-list/src/screens/UI/ButtonScreen.tsx

```diff
@@ -0,0 +1,147 @@
+import { H4 } from '@expo/html-elements';
+import { Button, ButtonProps } from '@expo/ui';
+import * as React from 'react';
+import { Platform, FlatList, ScrollView, View, StyleSheet } from 'react-native';
+
+import { Page } from '../../components/Page';
+
+function ButtonRenderItem(props: ButtonProps) {
+  return (
+    <Button
+      style={styles.buttonStyle}
+      onButtonPressed={() => {
+        console.log('Button pressed');
+      }}
+      {...props}
+    />
+  );
+}
+
+function ButtonDemoFlatList(props: { title: string; data: ButtonProps[] }) {
+  return (
+    <View style={styles.stretch}>
+      <H4>{props.title}</H4>
+      <FlatList
+        data={props.data}
+        renderItem={({ item }) => <ButtonRenderItem {...item} />}
+        scrollEnabled={false}
+        style={styles.stretch}
+        numColumns={2}
+        columnWrapperStyle={styles.columnWrapper}
+      />
+    </View>
+  );
+}
+
+const defaultButton: ButtonProps[] = [
+  {
+    text: 'Default button',
+  },
+];
+
+const buttonStylesData: ButtonProps[] = [
+  {
+    text: 'Default',
+  },
+  {
+    text: 'Bordered',
+    buttonStyle: 'bordered',
+  },
```

Comment:
Write the usage out explicitly instead of generating it from an array. This'll help us better visualize the way people will use the API, e.g. 
```js
<Button text="Bordered" buttonStyle="bordered" />
```

When visualized, it makes me think a more Reacty API would be something more like Radix:

```jsx
<Button variant="bordered">Bordered</Button>
```